mkdocs.yml_template

```diff
@@ -109,7 +109,6 @@ nav:
     - 'Contributing to Taipy': 'contributing/contributing.md'
     - 'Code of conduct': 'contributing/code_of_conduct.md'
   - 'Credits':
-    - 'Development lead': 'credits/development_lead.md'
```

Comment:
Why keep the 'Credits' title?
And not move this page under the 'Contributing' section. May be we can rename this...

Reply:
Yep I agree.

I let's move "Contributors" page to the 2nd page of the "Contributing" doc set (after "Contributing to Taipy" but before "Code of conduct").